builder/builder.go

```diff
@@ -217,11 +217,19 @@ func (b *Builder) convertAndBuild(ctx context.Context, target domain.Target, opt
 		// to accomodate parallelism in the WAIT/END PopWaitBlock handling
 		dirIDs = map[int]string{}
 	)
+
```

Comment:
nit: why not just combine the two `var` sections instead?

Reply:
I think it's ok to group var & const declarations by purpose or subject.